ChangeLog.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+### 2021-04-07 - 5.7.0
+
+min Autorest core version: 3.1.0
+
+min Modelerfour version: 4.15.456
+
+**New Features**
```

Comment:
Am kind of on the fence about making it a bug fix or a new feature. Ended up going with new feature so it's not as big a surprise when you get a new file